connectors/sources/notion.py

```diff
@@ -577,6 +602,9 @@ async def retrieve_and_process_blocks(self, query):
             yield self._format_doc(page_database), None
             self._logger.info(f"Fetching child blocks for block {block_id}")
 
+            if self.is_connected_property_block(page_database):
```

Comment:
Let's leave a debug log here before the `continue`

Reply:
Updated with commit id: [a0e3241](https://github.com/elastic/connectors/pull/2337/commits/a0e324129dad81dcbf0b949bfafa46ac79f73f55)